tests/configs/stanza-tokenizer-config.yml

```diff
@@ -0,0 +1,23 @@
+language: en
+
+pipeline:
+- name: rasa_nlu_examples.tokenizers.StanzaTokenizer
+  lang: "en"
+  cache_dir: "tests/data/stanza"
+- name: LexicalSyntacticFeaturizer
+  "features": [
+    ["low", "title", "upper"],
+    ["BOS", "EOS", "low", "upper", "title", "digit", "pos"],
+    ["low", "title", "upper"],
+  ]
+- name: CountVectorsFeaturizer
+  analyzer: char_wb
+  min_ngram: 1
+  max_ngram: 4
+- name: DIETClassifier
+  epochs: 1
+
+policies:
+  - name: MemoizationPolicy
+  - name: KerasPolicy
```

Comment:
Minor: I would use `TEDPolicy` instead of `KerasPolicy` as we are not really promoting `KerasPolicy` as far as I know.

Reply:
Nice one. I'm refactoring the tests now as is so I'll make this change in all configs.

Reply:
Come to think of it, I'm only testing NLU components here so I'll remove the policy parts alltogether.